components/dashboard/src/projects/new-project/NewProjectRepoSelection.tsx

```diff
@@ -172,18 +172,12 @@ export const NewProjectRepoSelection: FC<Props> = ({ selectedProvider, onProject
                             onSelectGitProvider={onChangeGitProvider}
                         />
                     )}
+                    <NewProjectSearchInput searchFilter={repoSearchFilter} onSearchFilterChange={setRepoSearchFilter} />
                     {enableBBSIncrementalSearch && (
                         <Alert type="info" className="text-sm -mx-4 my-2">
-                            <div>
-                                Repository search requires that search queries start with the first character(s) of the
-                                given repository name. Wildcard search is not supported.
-                            </div>
-                            <div className="mt-1">
-                                <strong>Example:</strong> For “my-repo” search “my-re”, instead of “repo”
-                            </div>
+                            <div>Bitbucket Server only supports prefix search.</div>
```

Comment:
**nitpick**: Minor style change and rephrase to be more explicit this affects only project search, as project-repository searching on /new works without needing prefix search, right?

```suggestion
                            <div>
                                <span className="font-semibold">Bitbucket Server</span> supports only prefix search for
                                projects.
                            </div>
```